snapshot-labs/snapshot-sidekick
# fix: update nft-claimer mint signing

## 🧿 Current issues / What's wrong ?

Signature implementation for NFT-Claimer is out-of-sync with the smart contract.

## 💊 Fixes / Solution

Update the NFT-Claimer signature schema to be in sync with what the smart contract is using.

## 🚧 Changes

- Update the signature schema
- Add tests

This PR only affect NFT-Claimer behavior, and can be pulled, as the signature is not live yet, and deploying this PR will allow easier testing by the SC team

## 🛠️ Tests

- Start the app 
- Run `curl -X POST localhost:3005/api/nft-claimer/mint/sign -H "Content-Type: application/json" -d '{"id": "0x28535f56f29a9b085be88e3896da573c61095a14f092ce72afea3c83f4feefe0", "address": "0x91FD2c8d24767db4Ece7069AA27832ffaf8590f3", "salt": "123456337495"}'`
- It should output something
